src/client/Microsoft.Identity.Client/AppConfig/ApplicationConfiguration.cs

```diff
@@ -134,6 +134,11 @@ public string ClientVersion
         public ITokenCacheInternal AppTokenCacheInternalForTest { get; set; }
 
         public IDeviceAuthManager DeviceAuthManagerForTest { get; set; }
+
+        public void SetAuthorityInfoForTest(AuthorityInfo authorityInfo)
+        {
+            AuthorityInfo = authorityInfo;

```

Comment:
Why is this needed? AuthorityInfo already has a setter at line 68.

Reply:
I think I added it because intellisense was telling me that it was not settable. I think it was probably just a bug. Should have verified